lua/metals/commands.lua

```diff
@@ -213,6 +213,16 @@ local commands_table = {
     label = "Update Metals",
     hint = "Update to the latest Metals.",
   },
+  {
```

Comment:
So one thing I was thinking about is right now these are unconditionally added. If a user doesn't have DAP setup they'll still see these. We probably don't want that right?